Networking/Networking/Network/AlamofireNetwork.swift

```diff
@@ -71,3 +75,57 @@ private extension Alamofire.DataResponse {
         }
     }
 }
+
+
+// MARK: - Alamofire.NetworkRequestRetrier
+//
+class NetworkRequestRetrier: RequestRetrier {
+
+    // [Request url: Number of times retried]
+    private var retriedRequests: [String: Int] = [:]
```

Comment:
Would it be necessary to make this property thread-safe, locking access to it when being modified?

Reply:
@ctarda that is a great point! I can refactor this to use a `NSRecursiveLock` or something similar.

Reply:
TIL about `NSRecursiveLock` 😊